inst/schema/metadata.json

```diff
@@ -93,6 +93,25 @@
             "items": {
               "$ref": "#/$defs/displayInfo"
             }
+          },
+          "time_series_roles": {
+            "type": "array",
+            "items": {
+              "$ref": "#/$defs/displayInfo"
+            }
+          },
+          "time_series_groups": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "id": { "type": "string" },
+                "label": { "type": "string" },
+                "time_series": { "type": "object" }
```

Comment:
Is there no way to specify that this object's keys must be 'time_series_roles'? If not, what good does it do to define the time_series_roles array?